### ISSUE TITLE
PagerDuty recipe does not install agent

### ISSUE THREAD

**Issue by jldugger** [sous-chefs/nagios#498]
`notify_pagerduty.pl` enqueues alerts in /tmp/pagerduty_nagios, but there's no agent installed to forward these on to PagerDuty.

https://www.pagerduty.com/docs/guides/agent-install-guide/ suggests they have a repo, GPG key and package. Is this functionality missing intentionally?

**Comment by mdarii** [comment]
@jldugger Hi, This cookbook is using perl script integration: https://www.pagerduty.com/docs/guides/nagios-perl-integration-guide/
It doesn't require to install PagerDuty agent. Perl script flush alerts to PagerDuty. If your nagios instance doesn't trigger alert in PD, check if on service/host pagerduty contact is added, also check if you are using right api key.

**Comment by jldugger** [comment]
Ah, thanks for that reference!

It looks like I need to add the pagerduty user to a contact group. I ran that perl script directly as root and didn't see anything go through, and couldn't find any logs for pdagent. As we're running CentOS7 and want bidirectional support, I hadn't really considered the perl script compatibility mode. Perhaps a separate setup for centos 6+ should be used.

`Jul 21 16:55:01 mon1.osuosl.org pagerduty_nagios[20555]: Nagios event in file /tmp/pagerduty_nagios/pd_1469035215_27159.txt DEFERRED due to network/server problems.`

I'll have to look into that more, but I at least understand more about the problem.

**Comment by jldugger** [comment]
Had to install `yum install perl-LWP-Protocol-https`, which isn't mentioned in PD's docs.

**Comment by jldugger** [comment]
tl;dr: CentOS7 support is not present, as clearly documented in the metadata.

**Comment by damacus** [comment]
@jldugger if you'd like to open a PR and fix this we'd greatly receive it! 

Thanks!
